Libraries/Components/DatePicker/DatePickerIOS.ios.js

```diff
@@ -97,6 +112,25 @@ var DatePickerIOS = React.createClass({
     };
   },
 
+  _onChangeCountDownDuration: function(event: Event) {
+    var nativeDate = new Date(event.nativeEvent.timestamp);
+    var nativeDuration = nativeDate.getMinutes() * 60 + nativeDate.getHours() * (60 * 60);
```

Comment:
Is there a better way of getting this? Like, maybe getting it from the Objective-C view?